### PR TITLE
Add a hello world example

### PR DESCRIPTION
**Pull request by lalithsuresh**
I didn't explain all the steps to use the current UI given its a placeholder anyway.

Signed-off-by: Lalith Suresh <lsuresh@vmware.com>

### CODE REVIEW

File: examples/orgchart/orgchart.rs [hunk 1]
```diff
@@ -0,0 +1,136 @@
+//! Simple DBSP example.
```
Comment (mbudiu-vmw) [inline examples/orgchart/orgchart.rs:1]: why is this file here?

Comment (lalithsuresh) [reply examples/orgchart/orgchart.rs]: Check again, messed up the git move.

File: examples/hello-world/README [hunk 2]
```diff
@@ -0,0 +1,18 @@
+First, build the DBSP docker container following the instructions
```
Comment (ryzhyk) [inline examples/hello-world/README:1]: `examples` is a special folder where cargo looks for when one runs `cargo run --example`.  We shouldn't abuse it for end-to-end examples.  The right thing to do is to reorganize the project moving all crates to a separate `crates` folder so we can have end-to-end `examples` here.  We can do this once other outstanding PRs have been merged. Maybe we can keep it with other projects in `pipeline_manager/demo` for now?
